### DISCUSSION TITLE
[Fixed] P-NP is not working.

### DISCUSSION THREAD

**Discussion by afkvido** [ProdigyPNP/ProdigyMathGameHacking#174] [Developer Forum]
Wtf is going on.

**Comment by eris-webserv** [comment]
Prodigy is loading *something* more than once, overwriting your hacks.
I won't tell you the something. That's your only hint :)

**Reply by afkvido** [reply]
![image](https://user-images.githubusercontent.com/69060894/192156864-1f5212fc-bfcb-4ab2-a956-eb12b4a89f25.png)

**Reply by afkvido** [reply]
Thanks for the hint I guess. I can probably just run `git diff` on the old and new gamefiles. That'l likely help me out tbh.

**Reply by eris-webserv** [reply]
> ![image](https://user-images.githubusercontent.com/69060894/192156864-1f5212fc-bfcb-4ab2-a956-eb12b4a89f25.png)

I was just watching this whole situation, found it hilarious, and gave you a hint as a joke.

**Reply by eris-webserv** [reply]
> Thanks for the hint I guess. I can probably just run `git diff` on the old and new gamefiles. That'l likely help me out tbh.

Good luck.

**Reply by afkvido** [reply]
xd a lot of prodigy's webpack stuff gets randomized so it's a bit fckin annoying

**Reply by hostedposted** [reply]
> xd a lot of prodigy's webpack stuff gets randomized so it's a bit fckin annoying

I'll give you a hint, I did not find out about this through the webpack bundle.

**Reply by hostedposted** [reply]
I may tell you what it is if you:

- Remove All Negative Things About Prodigy X from your discord server
- Unblock the keywords ProdigyX and rxzyx/prodigy-hack
- Make an announcement saying good stuff about Prodigy X
- Unmute me and unban AGR
- Have a link crediting Prodigy X on Prodigy PNP

**Reply by eris-webserv** [reply]
> I may tell you what it is if you:
> 
> * Remove All Negative Things About Prodigy X from your discord server
> * Unblock the keywords ProdigyX and rxzyx/prodigy-hack
> * Make an announcement saying good stuff about Prodigy X
> * Unmute me and unban AGR
> * Have a link crediting Prodigy X on Prodigy PNP

👍 🍿

**Reply by afkvido** [reply]
I could do that, but what's wrong with blocking `rxzyx/prodigy-hack`?

**Reply by afkvido** [reply]
And also, by "tell me what it is", do you mean tell me about `lodashObject`, or making a PR to fix it?

**Reply by hostedposted** [reply]
He made another prodigy hack. In fact, if he did not find ``Boot.prototype.game`` your cheat menu would not be working right now.

**Reply by hostedposted** [reply]
I'll tell you why it is not working.

**Reply by afkvido** [reply]
> He made another prodigy hack. In fact, if he did not find `Boot.prototype.game` your cheat menu would not be working right now.

Didn't Arcerion find it?

**Reply by hostedposted** [reply]
No

**Reply by hostedposted** [reply]
I DMed it to Arcerion.

**Reply by afkvido** [reply]
:trollface:

**Reply by hostedposted** [reply]
<img width="455" alt="image" src="https://user-images.githubusercontent.com/70211235/192352895-9499a6bc-d39f-4840-8a86-5de190acfa30.png">

**Reply by afkvido** [reply]
# 🗿 

oh

**Reply by afkvido** [reply]
@hostedposted @Erisfiregamer1 Alright, Alright, I'll do it. 
However, I'm asking that your PR the fix if I do.

Agree?

**Reply by hostedposted** [reply]
How do I know you will go through with this?

**Reply by eris-webserv** [reply]
I have an alt in their discord; I'll test. :)

**Reply by afkvido** [reply]
xd. So, agree?
